src/cmdc_tools/datasets/official/VA/data.py

```diff
@@ -0,0 +1,82 @@
+import pandas as pd
+
+from ...base import DatasetBaseNoDate
+from ..base import ArcGIS
+
+
+class Virginia(ArcGIS, DatasetBaseNoDate):
+    ARCGIS_ID = "BkFxaEFNwHqX3tAw"
+
+    def get(self):
+        df = self.get_all_sheet_to_df(service="VT_Counties_Cases", sheet=0, srvid=1)
+
+        renamed = df.rename(columns={
+            "CntyLabel": "county",
+            "CNTYGEOID": "fips",
+            "Cases": "cases_total",
+            "Deaths": "deaths_total",
+            "DateUpd": "dt"
+        })
+        # Convert Timestamps
+        renamed["dt"] = pd.to_datetime(renamed["dt"].map(
+            lambda x: pd.datetime.fromtimestamp(x/1000).date()
+        ))
+
+        county_summ = self._get_county_summary()
+
+
+        renamed = renamed[[
+            "county",
+            "fips",
+            "cases_total",
+            "deaths_total",
+            "dt"
+        ]]
+        result = pd.concat([renamed, county_summ], sort=False)
+        return (
+            result
+            .sort_values(['dt', 'county'])
+            .melt(
+                id_vars=['dt', 'county','fips'],
+                var_name="variable_name",
+                value_name="value"
+            )
+            .assign(
+                vintage=pd.Timestamp.utcnow().normalize()
+            )
+        )
+
+    def _get_county_summary(self):
+        df = self.get_all_sheet_to_df(service="county_summary", sheet=0, srvid=1)
+
+        renamed = df.rename(columns={
+            "date": "dt",
+            "positive_cases": "positive_tests_total", # daily
```

Comment:
What does `# daily` comment here mean? I believe we have defined `positive_tests_total` to be a cumulative total -- Is that what this reports?.

Reply:
"daily" means the number of new tests reported that day. I mentioned this discrepancy last week and asked what should be done. 

Some places only report new cases for that date, some only report total up to date, and some both.

I haven't gotten a concrete answer on a decision for what data should be kept/used